feats/feats.yml

```diff
@@ -839,19 +842,19 @@
   description: |
     Following a trail is an everyday part of your life, and as such it is no more taxing than breathing or blinking. Whether you are an experienced woodsman, a bounty hunter for the Galactic Senate, or a humanoid cat creature with heightened senses, your expertise at tracking allows you to get it done faster than the average hunter.
   effect: |
-    You can move at your full speed while tracking your quarry rather than the reduced speed typically incurred.
+    You move unhindered and at full speed when tracking your quarry.
 - !
   name: Ferocious Minions (I - III)
   prerequisites:
     tier1:
       Attribute:
-      - Influence: 3
+      - Invoking Attribute: 4
     tier2:
```

Comment:
I'm not familiar with the innerworkings of the site, but I think that it may work on a key-value approach, i.e it takes the key, combines it with the value and output the result on the screen. The `Any X` prerequisite would hint that way (i.e. it is not a valid attribute, it is a meta-attribute). That said, I too agree that this is a messy solution. Maybe something like `Creation, Entropy, Influence or Logic X` as a prerequisite would suit better.